### DISCUSSION TITLE
Change DATE format

### DISCUSSION THREAD

**Discussion by ChristKe** [Budibase/budibase#4635] [Help]
Hello everyone, 
I try to change the default date format **February 22, 2022** to look like  this 22/02/2022 or this 22-02-2022 using **Handlebars** at default value . 

Can someone please help me with an example?

**Comment by aelias36** [comment]
Should be `{{ date tableName.dateProperty "DD/MM/YYYY" }}` and ``{{ date tableName.dateProperty "DD-MM-YYYY" }}``

See here:
https://docs.budibase.com/building-apps/design/binding
https://momentjs.com/docs/#/displaying/format/

**Reply by ChristKe** [reply]
Thanks @aelias36, 
I tried both ways and still getting the default format, nothing changes... I even try to determine the time zone as  "Europe/Athens" after "DD-MM-YYY" but still...

**Reply by aelias36** [reply]
Strange - works for me.

![image](https://user-images.githubusercontent.com/4022499/155203566-cdc01726-5bca-47ba-b136-1d4e5a56d2fd.png)

**Reply by ChristKe** [reply]
I try to change the format inside a form not on paragraph... maybe this is the reason that doesn't work, idk.

https://share.icloud.com/photos/058qCbheOA8Tb8cU-ILcBaHHw

I copied exactly the code from your example.

**Reply by aelias36** [reply]
It looks like you can't currently change the format displayed in the date picker except for `Month day year, hours:minutes` or `Month day, year`

https://github.com/Budibase/budibase/blob/7cbbf56a93cc8aae550acdbd046e6967476447e5/packages/bbui/src/Form/Core/DatePicker.svelte#L26

**Reply by ChristKe** [reply]
Thank you very much for your help, really appreciate it.
